index.html

```diff
@@ -0,0 +1,468 @@
+                        <li>WhatsApp may let you set a self-destruct timer on private messages</li>                         
+                        <li>Microsoft launches a coronavirus tracker dashboard on Bing</li>                         
+                        <li>Instagram will place coronavirus advice front and center on your feed</li>                         
+                    </ul>                     
+                </article>                 
+            </div>             
+            <h1 class="latest-n">Latest Deals &gt;</h1> 
+            <div class="deals"> 
+                <a href="#"><article> 
+                        <div class="img-art2"></div>                         
+                        <h3>The Salesforce Certification Essensials Bundle</h3> 
+                        <p>$39.99</p> 
+                    </article> </a> 
+                <a href="#"><article> 
+                        <div class="img-art2"></div>                         
+                        <h3>The Salesforce Certification Essensials Bundle</h3> 
+                        <p>$39.99</p> 
+                    </article> </a> 
+                <a href="#"><article> 
+                        <div class="img-art2"></div>                         
+                        <h3>The Salesforce Certification Essensials Bundle</h3> 
+                        <p>$39.99</p> 
+                    </article> </a> 
+                <a href="#"><article> 
+                        <div class="img-art2"></div>                         
+                        <h3>The Salesforce Certification Essensials Bundle</h3> 
+                        <p>$39.99</p> 
+                    </article> </a> 
+                <a href="#"><article> 
+                        <div class="img-art2"></div>                         
+                        <h3>The Salesforce Certification Essensials Bundle</h3> 
+                        <p>$39.99</p> 
+                    </article> </a> 
+                <a href="#"><article> 
+                        <div class="img-art2"></div>                         
+                        <h3>The Salesforce Certification Essensials Bundle</h3> 
+                        <p>$39.99</p> 
+                    </article> </a> 
+                <a href="#"><article> 
+                        <div class="img-art2"></div>                         
+                        <h3>The Salesforce Certification Essensials Bundle</h3> 
+                        <p>$39.99</p> 
+                    </article> </a> 
+                <a href="#"><article> 
+                        <div class="img-art2"></div>                         
+                        <h3>The Salesforce Certification Essensials Bundle</h3> 
+                        <p>$39.99</p> 
+                    </article> </a> 
+            </div>             
+        </main>         
+        <footer class="main-footer"> 
```

Comment:
- [ ] The original footer takes 100% of the width of the screen. Check if you have some padding/margin issues in your body or other page elements.
- [ ] Pay attention to the bottom of the footer text. Can you make it more similar to the original?
**Original footer:**
![Screenshot_2020-03-22 TNW - International technology news, business and culture(1)](https://user-images.githubusercontent.com/16193212/77241703-cde81800-6bd4-11ea-8b34-36dd21181f0e.png)
**Your footer:**
![Screenshot_2020-03-22 The Next Web(3)](https://user-images.githubusercontent.com/16193212/77241704-ce80ae80-6bd4-11ea-8863-c1849ccceab9.png)